project/ConstantHolderGenerator.scala

```diff
@@ -33,12 +31,15 @@ object ConstantHolderGenerator {
     Seq(out)
   }
 
-  @tailrec
   private final def literal(v: Any): String = v match {
-    case s: String  => "raw\"\"\"" + s + "\"\"\""
+    case s: String  => "\"\"\"" + s + "\"\"\""
```

Comment:
I think the point of `raw` is to make sure that `\` coming from Windows paths are not interpreted. I believe it should be kept.

Reply:
Ugh... With `raw`, `$global` (from sbt) gets interpreted as a substitution... :( Is there another way to write strings that doesn't do that?

Reply:
Pass them through an `escapeScala` string that will put the appropriate escapes (and then it can just be a `""` string). Our existing `escapeJS` is probably good enough.